.github/workflows/tests.yml

```diff
@@ -9,24 +9,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ['2.7', '3.0', '3.1']
+        ruby: ['3.0', '3.1', '3.2']
```

Comment:
If we're dropping 2.7 from the build, we should update the [minimum required ruby](https://github.com/Betterment/demo_mode/blob/main/demo_mode.gemspec#L19) in the gemspec. (Or, we should keep 2.7 in the build matrix -- point being, if it's supported, it should be tested.)